unreleased/awsemfexporter-fix-4739.yaml

```
@@ -0,0 +1,5 @@
+change_type: bug_fix
+component: awsemfexporter
+note: "Properly handle empty dimension set in metric_declarations"
+issues:
+  - "#4739"
```

```suggestion
  - "#13766"
```

I prefer to link to the PR effecting the change.  This makes it easier for people who want to see what has changed to get the details.  It should in turn link to the original issue.

---

makes sense. thanks for the heads up. I updated the changelog.